api.json

```diff
@@ -0,0 +1,761 @@
+          },
+          "response": {
+            "-1": "Number",
+            "0": "Number",
+            "1": "Number",
+            "since": "String",
+            "until": "String"
+          }
+        },
+        "POST": {
+          "description": "Send feedback for one organisation",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "feedbackScore": "Number"
+          },
+          "response": {
+            "feedbackScore": "Number",
+            "organisationId": "Number"
+
+          }
+        }
+      },
+      "/:id/users": {
+        "GET": {
+          "description": "Retreive all users in an organisation",
+          "auth": {},
+          "scope": {},
+          "query": {
+            "page": "Number",
+            "offset": "Number",
+            "sort": "String",
+            "gender": "String",
+            "age": "String",
+            "emailConsent": "Boolean",
+            "smsConsent": "Boolean"
+          },
+          "response": [{
+            "id": "Number",
+            "name": "String",
+            "gender": "String",
+            "birthYear": "String",
+            "email": "String",
+            "phoneNumber": "String",
+            "emailConsent": "Boolean",
+            "smsConsent": "Boolean"
+          }]
+        }
+      },
+      "/:id/visits": {
+        "GET": {
+          "description": "Retreive all visit data for an organisation",
+          "auth": true,
+          "scope": {},
+          "query": {
+            "fields": ["String"],
+            "sort": "String",
+            "page": "Number",
+            "offset": "Number",
+            "gender": "String",
+            "age": "String",
+            "activity": "String"
+          },
+          "response": {}
+        }
+      },
+      "/:id/volunteers": {
+        "GET": {
+          "description": "Retrieve volunteer details for a single organisation",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "fields": ["String"],
+            "filter": {
+              "since": "String",
+              "until": "String"
+            }
+          },
+          "response": {
+            "totalVolunteers": "Number",
+            "totalVolunteerHours": "Number",
+            "volunteerLogs": [{
+              "volunteerLogId": "Number",
+              "userId": "Number",
+              "volunteerActivity": "String",
+              "duration": "Number",
+              "loggedAt": "String"
+            }]
+          }
+        }
+      },
+      "/:id/outreach": {
+        "GET": {
+          "description": "Retrieve all outreach campaings and meetings for a specific organisation",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "fields": ["String"],
+            "filter": {
+              "outreachType": "String"
+            }
+          },
+          "response": {
+            "result?": [{
+              "outreachCampaignId": "Number",
```

Comment:
Other responses use simply `id` as the key name of the top level entity. We should be consistent, and either use the full id key name everywhere or just use `id` everywhere.